### ISSUE TITLE
match_array with single element does not match Rspec behaviour

### ISSUE THREAD

**Issue by Mongey** [splitwise/super_diff#97]
This is valid when super_diff is not installed.

```ruby
  expect(ids).to match_array(single_id)
```
With super_diff installed it fails with
```
Failure/Error: expect(ids).to match_array(single_id)

NoMethodError:
  undefined method `each_with_index' for "my_id":String
  Did you mean?  each_line
```

I had to update the test to 
```diff
-        expect(ids).to match_array(single_id)
+        expect(ids).to match_array([single_id])
```

**Comment by mcmire** [comment]
Okay, thanks! I'll check it out and see what's up here.

**Comment by mcmire** [comment]
This is fixed in 0.5.3+! Closing.
